src/views/components/EntityCard.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+	<div class="entity-card" :style="getCardStyle()">
+		<img class="img-tier" 
+			:class="{'user-tier': !this.isOpponent(), 'opponent-tier': this.isOpponent()}" 
```

Comment:
when in moustache templates you don't need to do this.isOpponent, you should be able to do isOppenent() or probably even isOpponent

Reply:
I confirmed that you can remove 'this' and '()'.

I have left the parens, mainly just because I want to know at a glance if we're referencing a variable or a function.